pvHelpers/__init__.py

```diff
@@ -10,6 +10,7 @@
 from .hook_decorators import *
 from .application_config import ApplicationConfig
 from .proxy_config import *
+from .connection_info import *
```

Comment:
this is not being used right given https://github.com/PreVeil/core/pull/1649/files#diff-c939b7c062708b34ff1a3098758416cdR1379 is commented out?

Reply:
coorect